build.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
 	dependencies {
 		classpath "org.xtext:xtext-gradle-plugin:$versions.xtext_gradle_plugin"
 		classpath "io.spring.gradle:dependency-management-plugin:$versions.dependency_management_plugin"
+		classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:4.2.0'
```

Comment:
i think it would be better to move the 4.2.0 to a contant in versions.gradle

Reply:
Done